packages/forma-36-react-components/src/components/EditorToolbar/EditorToolbarButton/README.mdx

```diff
@@ -0,0 +1,62 @@
+import { Button } from '../../Button';
+import { EditorToolbar } from '../EditorToolbar';
+import { EditorToolbarButton } from './EditorToolbarButton';
+
+# EditorToolbarButton
+
+The EditorToolbarButton is a styled button for actions that should be shown on a rich text editor toolbar. It composes the IconButton component, which means it can show any icon that the IconButton can.
+
+<EditorToolbar>
```

Comment:
@denkristoffer same here - I would move it below the Examples of usage (and adjust the title of the section)